model/lifecycle.go

```diff
@@ -522,6 +522,16 @@ func CreateBuildFromVersion(project *Project, v *version.Version, taskIds TaskId
 	if err != nil {
 		return "", errors.Wrapf(err, "error creating tasks for build %s", b.Id)
 	}
+	tasksToLog := []string{}
+	for _, t := range tasksForBuild {
+		tasksToLog = append(tasksToLog, t.Id)
+	}
+	grip.Debug(message.Fields{
+		"ticket":  "EVG-5226",
+		"version": v.Id,
+		"variant": b.BuildVariant,
+		"tasks":   tasksToLog,
```

Comment:
Do you really need the tasks, or should you log their length instead? (Either is fine, though I wonder if with a very large number of tasks, logging may be awkward or get dropped, especially if this gets hit a lot.)